example/Makefile

```diff
@@ -1,41 +1,63 @@
+ifeq ($(shell which llvm-config-12),)
+  ifeq ($(shell which llvm-config-9),)
+    ifeq ($(shell which llvm-config),)
+      $(error Unable to locate the LLVM toolchain. \
+        Check your $$PATH or reinstall. Supported LLVM versions are 9 and 12)
+    else
+      LLVM_CONFIG = llvm-config
+      CLANG_FORMAT = clang-format
```

Comment:
In Linux, I believe clang-format is downloaded separately than the rest of LLVM toolchain. I'd prefer if we just use clang-format without the version prefix, always, as long as it works fine on Darwin too.

Reply:
I've made changes to use `clang-format` without the version suffix. We also prioritize `clang-format` with version suffix for compatibility reasons if we can find it. If we can't find any `clang-format` binaries at all, we will ignore the error instead of stopping. Updated in b45282c

Reply:
I've made changes to use `clang-format` without the version suffix. We also prioritize `clang-format` with version suffix for compatibility reasons if we can find it. If we can't find any `clang-format` binaries at all, we will ignore the error instead of stopping. Updated in b45282c.